### ISSUE TITLE
Show value as int or float

### ISSUE THREAD

**Issue by gerard33** [m1n3rva/bimmer_connected#68]
Currently the values in `state.py` are shown as float of int, eg:
```
def mileage(self) -> float:
def max_range_electric(self) -> int:
```
resulting in:
![screenshot_5](https://user-images.githubusercontent.com/11230573/38777780-35a8ce20-40ae-11e8-86de-8bd62ac6b7bd.png)

I would prefer to have these values all as int, but as long as these are shown consistent it's fine by me :grinning: 

Also in some cases the `get` method is used:
`return self._attributes.get('remainingRangeElectric')`
and in some cases not:
`return float(self._attributes['mileage'])`

@m1n3rva shall I make a PR for this?

**Comment by m1n3rva** [comment]
Yes, you're right, this should be consistent, fixed in #70

**Comment by gerard33** [comment]
@m1n3rva 
I also saw some other differences.

> Also in some cases the get method is used:
return self._attributes.get('remainingRangeElectric')
and in some cases not:
return float(self._attributes['mileage'])

Should that be made consistent as well?

**Comment by m1n3rva** [comment]
@gerard33 good point, I changed this.

**Comment by gerard33** [comment]
Fixed in PR #70.
